torchmetrics/text/wer/wer.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from jiwer import compute_measures
+
+from torchmetrics.metric import Metric
+
+
+class WER(Metric):
+
+    def __init__(self, concatenate_texts=False):
+        super().__init__()
+        self.concatenate_texts = concatenate_texts
+
+    def update(self, preds: Any, target: Any) -> None:
+        self.preds = preds
+        self.target = target
```

Comment:
i assume that we want to accumulate over multiple sentences
```suggestion
        self.preds.append(preds)
        self.target.append(target)
```

Reply:
Yes you are right